modal-pop-up.js

```diff
@@ -0,0 +1,213 @@
+// const body = document.querySelector('body');
+// const overlay = document.querySelector('.overlay');
+// const mainBg = document.querySelector('.main-bg');
+// const buttons = document.querySelectorAll('.see-projects-btn');
```

Comment:
- [ ] I'll suggest you remove all the commented code you have in your project, as leaving a commented piece of code in your project makes it look unprofessional, messy and it's against the best practices. 😄 👍